tests/functional/013-expected-stderr/functional/source/data/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "code":1,
+  "stderr": "Something fails"
```

Comment:
zkus tady přidat `\n` na konec toho stderr - to by se klidně mohlo stát, že tan výstup bude vypadt takhle..

Reply:
a to je dobre nebo spatne?

Reply:
No, je jedno jak to je. Ale stát se to může. Druhá věc je, jak se to má chovat a jestli současné chování je správně nebo ne :)

Reply:
teď když je tam `assertContains` tak je to jedno.